components/Banner/AnimatedLineartBanner.tsx

```diff
@@ -25,7 +25,7 @@ export const AnimatedLineartBanner = ({
       <HorizontalLineart
         stroke={stroke}
         lineartType={lineartType}
-        className="ml-auto rs-pl-5 md:rs-pl-8 max-w-[160rem] 2xl:max-w-[200rem] 3xl:max-w-[250rem] 3xl:rs-pr-5"
```

Comment:
I try to avoid doing this these days `rs-pl-5 md:rs-pl-8` since it doubles up on the media queries. I might have used it in the past myself but I have stopped using it now.

Reply:
Oh, I wonder if there is a way to make these utilities smarter in Tailwind where if you're adding the breakpoint it pulls the breakpoint value instead of doubling the media query...